README.md

```diff
@@ -11,10 +11,36 @@ for details on the library's purpose and functionality.
 Installing
 ----------
 
-To install, first make sure you have [Go](https://golang.org)
-version 1.3 or newer installed, then run:
+First make sure you have [Go](https://golang.org)
+version 1.3 or newer installed.
+
+Install C library dependencies.
+
+OpenSSL: [OpenSSL](https://www.openssl.org/) 
+
+Pairing-Based Cryptographay: [PBC](http://crypto.stanford.edu/pbc/)
```

Comment:
Also, PBC is actually optional and not actually needed by anything in the crypto library we're currently using, so maybe we should add a note to that effect.  (All that will happen is that directory will fail to build/test.)  We actually need to figure out the "right" way to deal with optional dependencies like this in the Go environment; let me know if you run across any relevant wisdom.